### PR TITLE
Provide an easier way to run collect_env

### PR DESCRIPTION
**Pull request by ppwwyyxx**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ Please copy and paste the output from our
 [environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
 (or fill out the checklist below manually).
 
-You can get the script and run it with:
+If PyTorch is successfully installed, run `python -m torch.utils.collect_env`. 
+Otherwsie, you can get the script and run it with:
```
Comment (fmassa) [inline .github/ISSUE_TEMPLATE.md]: nit: Otherwise

**Comment by ppwwyyxx** [comment]
Found that there are other templates https://github.com/pytorch/pytorch/blob/master/.github/ISSUE_TEMPLATE/bug-report.md. Will change them in a moment

**Pull request closed without merging**